mechanisms/mod/hh.mod

```diff
@@ -47,25 +48,24 @@ BREAKPOINT {
 }
 
 INITIAL {
-    rates(v, celsius)
+    q10 = 3^((celsius - 6.3)/10)
+    rates(v)
     m = minf
     h = hinf
     n = ninf
 }
 
 DERIVATIVE states {
-    rates(v, celsius)
+    rates(v)
     m' = (minf-m)/mtau
     h' = (hinf-h)/htau
     n' = (ninf-n)/ntau
 }
 
-PROCEDURE rates(v, celsius)
+PROCEDURE rates(v)
 {
     LOCAL  alpha, beta, sum, q10
```

Comment:
This local variable `q10` is hiding the new assigned variable, and is used instead.
As a result, there is a warning during compilation, and one of the unit tests is not passing.

Reply:
Fixed, tests are ok.